mhkit/tidal/io/noaa.py

```diff
@@ -124,7 +124,10 @@ def request_noaa_data(
 
     # Use handle_caching to manage cache
     cached_data, cached_metadata, cache_filepath = handle_caching(
-        hash_params, cache_dir, write_json=write_json, clear_cache_file=clear_cache
+        hash_params,
+        cache_dir,
+        cache_content={"data": None, "metadata": None, "write_json": write_json},
+        clear_cache_file=clear_cache,
```

Comment:
handle_caching reduced to only 5 inputs
https://pylint.readthedocs.io/en/stable/user_guide/configuration/all-options.html#max-args